scripts/joinmarket-qt.py

```diff
@@ -328,20 +329,24 @@ def __init__(self):
         self.spendstate.reset() #trigger callback to 'ready' state
 
     def generateTumbleSchedule(self):
+        if not w.wallet:
```

Comment:
s/w/mainWindow/

Reply:
But also, good addition here, it was needed to have this check, just update the names to `mainWindow` and `wallet_service`.